site/app.js

```diff
@@ -1,45 +1,60 @@
-require('dotenv').config({ silent: true })
+require("dotenv").config({ silent: true })
 
-const express = require('express');
-const onHeaders = require('on-headers')
-const app = express();
-const csp = require('./app/csp').default;
-const hstsheader = require('./app/hstsheader').default;
+const CustomStorage = require("./app/uploadcare/customStorage")
+const UploadcareApi = require("./app/uploadcare/api")
+const express = require("express")
+const onHeaders = require("on-headers")
+const app = express()
+const csp = require("./app/csp").default
+const hstsheader = require("./app/hstsheader").default
 
 const SP_PROTECTED = (process.env.SP_PROTECTED || "true").toLowerCase()
+const FEATURE_UPLOADCARE_IS_ENABLED = (
+  process.env.GATSBY_FEATURE_UPLOADCARE_IS_ENABLED || "false"
+).toLowerCase()
 
-const withoutEtag = (response) => {
+const withoutEtag = response => {
   onHeaders(response, function() {
-    this.removeHeader('ETag')
+    this.removeHeader("ETag")
   })
   return response
 }
 
-app.use(csp({
-  chatDomain : process.env.GATSBY_CHAT_DOMAIN,
-  analyticsHost : process.env.MATOMO_IP
-}));
+app.use(
+  csp({
+    chatDomain: process.env.GATSBY_CHAT_DOMAIN,
+    analyticsHost: process.env.MATOMO_IP,
+    imageBucket: "https://*.amazonaws.com"
```

Comment:
How does this work with the wildcard?

Reply:
Yeah, images don't load on the site without it